Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1670,7 +1670,7 @@ def send_mail(emailto, emailfrom, subject, body, entry_ids, cc, bcc, htmlBody, r
         # if there is only body and no attachments to the mail , we would like to send it without attaching every part
         message = MIMEText(body, 'plain', 'utf-8')  # type: ignore
     else:
-        message = MIMEMultipart()  # type: ignore
+        message = MIMEMultipart('alternative') if body and htmlBody else MIMEMultipart()  # type: ignore
```

Comment:
what happens if we just set it to `MIMEMultipart('alternative')` if not both provided ?

Reply:
I'll test it now.

Reply:
Still seems to work. I tested it with,
1. `body` and `htmlBody`,
2. just `body`,
3. just `htmlBody`.
Regarding all the other command arguments I haven't checked.

Reply:
ok lets keep it as is